spec/project/project_spec.rb

```diff
@@ -294,6 +297,12 @@
       expect(@project.user_groups(new_group)).not_to be_nil
       expect(@project.user_groups(new_group).member?(u.uri)).to be_truthy
     end
+
+    after(:all) do
+      other_user, *users = ProjectHelper.ensure_users(client: @client, amount: 3, caller: $example_name)
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `other_user`. Use `_` or `_other_user` as a variable name to indicate that it won't be used.